Source/WebKit/GPUProcess/webrtc/RemoteAudioMediaStreamTrackRendererInternalUnitManager.cpp

```diff
@@ -72,7 +72,7 @@ class RemoteAudioMediaStreamTrackRendererInternalUnitManager::Unit
     void storageChanged(SharedMemory*, const WebCore::CAAudioStreamDescription&, size_t);
 
     // CoreAudioSpeakerSamplesProducer
-    const WebCore::CAAudioStreamDescription& format() final { return m_description; }
+    const WebCore::CAAudioStreamDescription& format() final { return *m_description; }
```

Comment:
That seems a bit dangerous, maybe we should remove format() virtual method and try to pass the CAAudioStreamDescription directly in registerSpeakerSamplesProducer.